src/cli.ts

```diff
@@ -404,13 +399,16 @@ export const buildCmd = (): CompletableCommand => {
     .description("Push local YAML test cases to the test target")
     .helpGroup("test-cases")
     .addOption(testTargetIdOption)
-    .option(
-      "-s, --source <path>",
-      "Source directory (defaults to current directory)",
-      ".octomind",
-    )
     .action(addTestTargetWrapper(pushTestTarget));
 
+  createCommandWithCommonOptions(program, "show-octomind-dir")
```

Comment:
i wonder if this should just be part of every command? Maybe fine like this to start.


(then we might have to cache it not sure how expensive it is 😬 )